### ISSUE TITLE
[a11y] scroll bar is missing role

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#161559]
When using this widget https://api.flutter.dev/flutter/material/Scrollbar-class.html and https://api.flutter.dev/flutter/cupertino/CupertinoScrollbar-class.html, the widget is missing following role

| OS | role |
|--------|--------|
| web | scrollbar |
| ios | - |
| macos | NSAccessibilityScrollBarRole |
| windows | ROLE_SYSTEM_SCROLLBAR |
| android | - |
